core/backend/package.json

```diff
@@ -101,6 +101,8 @@
   "dependencies": {
     "@bentley/imodeljs-native": "5.0.41",
     "@itwin/cloud-agnostic-core": "^2.2.4",
+    "@itwin/ecschema-locaters": "workspace:*",
```

Comment:
doesnt seem to be used

Reply:
Fair point, I just added this because I know we will need it eventually. This PR is supposed to be a minimal "get the foot in the door" one, and we will iterate on it, where we will make use of schema locaters.
Schema Registry for example uses global paths to schemas on the file system, which I have not rewired to use schema locaters yet, but I will eventually.
Can we still add it in this PR already? I feel those two packages sort of belong together.

Reply:
I would prefer to remove just in case we don't end up using it and accidentally release with it. But as long as we are 100% confident we will be using it, its fine.

Reply:
Yeah sounds reasonable. I am not 100% confident we're going to use it, so I removed the dependency for now